exporter/redisPwd.go

```diff
@@ -0,0 +1,25 @@
+package exporter
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var RedisPwd map[string]string
+
+func LoadPwdFile(pwdFile string) error {
```

Comment:
This shouldn't set a global variable but return the passwords read from the file.

You could declare a type called PasswordMap and then return this here along with the error.